### PR TITLE
Improve url selection

### PR DESCRIPTION
**Pull request by vallsv**
Changelog: 
- `UrlSelection`:
    - Rework the look and feel
    - Allow to drop an URL from `silx view`
    - Fixes typos on variable names

### CODE REVIEW

**Comment by vallsv** [comment]
This is ready to review

**Review by payno — APPROVED** [review]
Nice improvement great.

File: src/silx/gui/widgets/UrlSelectionTable.py [hunk 1]
```diff
@@ -185,6 +185,20 @@ def addUrl(self, url, **kwargs):
...
+            if url.path() == urlPath:
+                break
+        else:
+            return
+
+        style = qt.QApplication.style()
+        icon = style.standardIcon(qt.QStyle.SP_MessageBoxCritical)
+        item.setIcon(icon)
+        item.setToolTip(f"Error: {message}")
+
```
[the remark this answers is not in the document]
Comment (vallsv) [reply src/silx/gui/widgets/UrlSelectionTable.py]: Yes, it's usually better to do that, even more to to put that in the model.
Ill do that.

**Comment by vallsv** [comment]
Here is an update with some tests

**Review by t20100 — APPROVED** [review]

**Pull request merged**